torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -654,9 +662,9 @@ at::Tensor PythonArgs::tensor_slow(int i) {
     scalar = at::Scalar(THPUtils_unpackBool(obj));
   } else if (THPUtils_checkLong(obj)) {
     scalar = at::Scalar(THPUtils_unpackLong(obj));
-  }else if (PyComplex_Check(obj)) {
+  } else if (PyComplex_Check(obj)) {
     scalar = at::Scalar(THPUtils_unpackComplexDouble(obj));
-  }else if (THPUtils_checkDouble(obj)) {
+  } else if (THPUtils_checkDouble(obj)) {
```

Comment:
There's a callsite in tuple_parser.cpp that we might have to change. I'm not sure what it does though but it would be good to figure out what it is doing and how this PR affects it:
https://github.com/pytorch/pytorch/blob/e6000a7c045cbece5fbfd7d933c39e40b1625037/torch/csrc/utils/tuple_parser.cpp#L42

Reply:
hmmm... I think you're probably right but I'm not seeing any calls to this function that I could use to validate expected behavior. Do you think it would be alright to just remove tuple_parser?

I see one use for string parameters in torch/csrc/autograd/functions/init.cpp, but I think a small modification there would allow me to eliminate the rest of this tuple_parser code.

Reply:
for now I'm reverting the change to that function, which (assuming the tests pass this way) bypasses the issue. I'm still tempted to remove tuple_parser in a subsequent PR though if that makes sense.